sdk/lib/3rdparty/freetype/src/autofit/afmodule.c

```diff
@@ -545,18 +523,16 @@
 
     af_loader_done( loader );
 
+#ifdef __REACTOS__
+    free(loader);
+#endif
     return error;
 
 #else /* !FT_DEBUG_AUTOFIT */
 
 #ifdef __REACTOS__
-    AF_GlyphHintsRec *hints = malloc(sizeof(AF_GlyphHintsRec));
-    AF_LoaderRec *loader = malloc(sizeof(AF_LoaderRec));
-    if (!hints || !loader)
-    {
-        error =  FT_Err_Out_Of_Memory;
-        goto Exit;
-    }
+    AF_GlyphHintsRec *hints = malloc(sizeof(*hints));
+    AF_LoaderRec     *loader = malloc(sizeof(*loader));
```

Comment:
Please restore the NULL checks as was done in the previous ROS code version.

Reply:
Restored.